fossunited/payments/doctype/razorpay_payment/razorpay_payment.json

```
@@ -137,20 +137,21 @@
       "fieldname": "gstn",
       "fieldtype": "Data",
       "label": "GSTN",
-      "read_only": 1
+      "permlevel": 1,
+   "read_only": 1
```

nit - weird formatting, might be a bug in the autogeneration

```suggestion
      "permlevel": 1,
      "read_only": 1
```

---

💯

---

I think the formatting is a bit off for every json file. Can you check @RiyaMathew-11 ? Make sure you run prettier with the latest settings of repo

---

I've run prettier on the file again. Looks ok now